JustBrewIt/Pages/Search.cshtml

```diff
@@ -1,17 +1,18 @@
 ﻿@page
 @model JustBrewIt.Pages.SearchModel
 @{
+    ViewData["Title"] = "Search";
     var BreweryAPI = (JustBrewItQuickType.Welcome[])ViewData["BreweryAPI"];
 }
 
 <form action="" method="POST" style="margin-top: 50px;">
     <div class="form-group">
-        <label class="control-label" for="BreweryCity">Search Brewries by City</label>
+        <label class="control-label" for="BreweryCity">Search Breweries by City in Ohio</label>
```

Comment:
What if there are breweries in another state?  Now or in the future?  You'd have to make a code change.  I'd leave this more general.